babakhani/persian-datepicker-next-version
# feat: add e2e tests - continues dates

- Add cypress
- Add e2e test scenario:
  - Go to 1300 and select Farvardin-1300
  - Check there is no duplicate date in the month
  - Go to next month until 1500 and check previous step for every month

How to run:  
- Run `yarn dev`
- Run `yarn cypress` (this takes too much time, you can see progress by `npx cypress open`)

System requirements:  
https://docs.cypress.io/guides/getting-started/installing-cypress.html#System-requirements
